src/guid.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Unique ID for an element or function
+ * @type {Number}
+ */
+let _guid = 1;
```

Comment:
No need to prefix this as it's inherently private by virtue of being inside a module.